### PR TITLE
Fixes and debugging

### PR DESCRIPTION
**Pull request by andrei-livadariu**
This is a pull request with small fixes based on your feedback and some changes for debugging.

1. The Points class is now using operator overloading. Additionally, points are now being displayed to users as integers. They don't see any more decimals, although decimals are still used behind the scenes. E.g. if I have 2.7 points I will see 2 points in my balance but I can actually donate up to 2.7 points. This seems fair.
2. I added some logging to modules, concentrating on the birthday module to figure out what's wrong with it. The default log level has been set to INFO instead of DEBUG, so we have fewer messages to go through. It can also be set through an environment variable so we always have the option to increase log verbosity, even in production. Also, I temporarily added a command that forces birthdays to be displayed to the chat. This might help me with debugging. I'll delete it afterwards.
3. The Loyverse functionality has been moved to the folder integrations/loyverse . It has been split into several files and the LoyverseConnector class has been renamed to LoyverseApi . In the next pull request I will have another integration, this time with Google, so there is a similar pattern here.

### CODE REVIEW

File: helpers/points.py [hunk 1]
```diff
@@ -13,6 +13,33 @@ def __init__(self, amount: Union[str, int, float, Decimal]):
         except (InvalidOperation, ValueError) as error:
             raise UserFriendlyError('The number of points must be ... a number 😬') from error
 
+    def __add__(self, other) -> "Points":
```
Comment (TheFreshPenguin) [inline helpers/points.py:16]: niiiiiiiice

File: helpers/points.py [hunk 2]
```diff
@@ -22,11 +49,8 @@ def is_negative(self) -> bool:
     def is_zero(self) -> bool:
         return self.amount.is_zero()
 
-    def add(self, delta: "Points") -> "Points":
-        return Points(self.amount + delta.amount)
-
-    def subtract(self, delta: "Points") -> "Points":
-        return Points(self.amount - delta.amount)
+    def to_integral(self) -> "Points":
```
Comment (TheFreshPenguin) [inline helpers/points.py:52]: lmao you could have just do a int(points) i believe

File: integrations/loyverse/customer.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+
+from helpers.points import Points
+
+
+class Customer:
```
Comment (TheFreshPenguin) [inline integrations/loyverse/customer.py:6]: forte bine!!!!!!!!!!!!!!! veri nice

**Review by TheFreshPenguin — APPROVED** [review]
okok

File: helpers/points.py (hunk 2 above)
Comment (andrei-livadariu) [reply helpers/points.py:52]: You're right, but since Decimal has all sorts of weird rounding rules I wanted to make the rounding explicit.

**Pull request merged**